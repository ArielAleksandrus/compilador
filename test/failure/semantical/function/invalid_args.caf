int soma(int a, int b){
	retorne a + b;
}

programa{
	int a = 3;
	soma(a, 'b');
}