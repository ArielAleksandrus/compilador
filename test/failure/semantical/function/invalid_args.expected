ERROR => Line 7, col 5: Function declaration of soma(INT, CAR) not found
